integration/nwo/network.go

```diff
@@ -633,6 +647,9 @@ func (n *Network) GenerateConfigTree() {
 	for _, p := range n.Peers {
 		n.GenerateCoreConfig(p)
 	}
+	for _, cc := range n.ChaincodeServers {
+		n.GenerateChaincodeConnectionConfig(cc)
+	}
```

Comment:
This should not be part of the network model; these are generated by the external builder implementations.